packages/typespec-ts/test/unit/apiVersion.spec.ts

```diff
@@ -1,6 +1,6 @@
 import { assert } from "chai";
-import { emitClientFactoryFromCadl } from "./util/emitUtil.js";
-import { assertEqualContent } from "./util/testUtil.js";
+import { emitClientFactoryFromCadl } from "../util/emitUtil.js";
```

Comment:
Please create an issue for MQ that we need to replace Cadl with TypeSpec/tsp in our codebase.

Reply:
updated in this PR. no need to create an additional issue.